### DISCUSSION TITLE
dalle-mini with ludwig?

### DISCUSSION THREAD

**Discussion by fire** [ludwig-ai/ludwig#2166] [Ideas]
Since it's possible to use any Hugging face model for a text prompt to image output.

Do you think it's possible to use dalle-mini?

**Comment by w4nderlust** [comment]
@fire at this very moment we do not have an image output feature implementation, but i believe it would be relatively straightforward to add. In that case you could have a text input and image output both loading from something like dalle-mini or HF.

I would consider this a feature request and something that some contributor may find interesting to work on!
